python/ray/node.py

```diff
@@ -190,6 +185,19 @@ def __init__(self,
             self._raylet_socket_name = self._prepare_socket_file(
                 self._ray_params.raylet_socket_name, default_prefix="raylet")
 
+        self.metrics_agent_port = self._get_cached_port(
+            "metrics_agent_port", default_port=ray_params.metrics_agent_port)
+        self._metrics_export_port = self._get_cached_port(
+            "metrics_export_port", default_port=ray_params.metrics_export_port)
+
+        self._metrics_export_port = ray_params.metrics_export_port
```

Comment:
We don't need this block right?

Reply:
Oh, didn't we say that whatever approach we do for metrics_agent_port we should do for metrics_export_port too? I'm not fully familiar with the second one yet so it's your call. Also, I'll do the rename of metrics_agent to dashboard_agent in a separate PR so that this diff is easier to read.

Reply:
You already assigned a port of this in line 190 right? Am I missing something?

Reply:
Line 193-195 will make line 190 meaningless IIUC?

Reply:
Ah I see, you're totally right, I forgot to remove lines 193-195.  I'll do that